Documentation/Building.md

```diff
@@ -120,6 +169,26 @@ EVENT__ENABLE_VERBOSE_DEBUG:BOOL=OFF
 # documentation for try_run for more details
 EVENT__FORCE_KQUEUE_CHECK:BOOL=OFF
 ```
+MSVC specific options:
```

Comment:
I always wanted to drop them and just leave the following command `cmake -LAH | fgrep EVENT__ -B1`, but I guess that not all are ready to clone and run this command, so just ignore this comment